packages/material-ui/src/Select/SelectInput.js

```diff
@@ -369,6 +369,20 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
       foundMatch = true;
     }
 
+    const hasOptionSelected = () => {
```

Comment:
And here, when you select an option don't need more to set the second option as selected true to gain the focus.